### PR TITLE
Updated Ember-Frost-Core@^1.0.0

### PR DESCRIPTION
**Pull request by NickLewanowicz**
Section in the demo css was causing a visual bug, once removed everything worked perfectly. All functionality is maintained.

### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [x] #major# - incompatible API change

# CHANGELOG
- All functionality was maintained

### CODE REVIEW

File: tests/dummy/app/styles/app.scss [hunk 1]
```diff
@@ -40,11 +40,6 @@ fieldset {
 
     .field-value {
       width: 250px;
-
-      >div,
```
Comment (dafortin) [inline tests/dummy/app/styles/app.scss]: `> div`

**Comment by dafortin** [comment]
Can you update the dependencies please: https://www.bithound.io/github/ciena-frost/ember-frost-notifier/ee253882c44964e30ec06070171e135ad9bb4278/dependencies/npm#filter-failing-dep

**Comment by rox163** [comment]
You should be able to remove ember-cli-app-version

File: package.json [hunk 2]
```diff
@@ -31,15 +31,14 @@
...
-    "ember-cli-app-version": "^1.0.0",
     "ember-cli-code-coverage": "~0.3.0",
     "ember-cli-content-security-policy": "^0.5.0",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-ecosystem-installer": "2.0.25",
     "ember-cli-ember-ecosystem": "0.4.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mocha": "^0.10.0",
+    "ember-cli-mocha": "~0.13.0",
```
Comment (rox163) [inline package.json]: Cause of your build failure - chai is now provided by the `ember-cli-chai` addon which will be added to your projects by default during ember install ember-cli-mocha.
If you upgrade your version of ember-cli-mocha you will have to run ember generate ember-cli-mocha to make sure the dependency gets installed in your project!

**Pull request merged**